styles/components/app-nav.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 
 <nav class="nav-container">
-  <ul class="nav-container__list">
+  <ul class="nav-container__list" role="list">
```

Comment:
is it really necessary to give an `ul` a `role="list"`??? in my opion a unordered list is already a list!?

Reply:
hi, i added this because of accessibility - if i understood our head-coach correctly, `role="list"`is then necessary when you have `liststyle: none` because then a screenreader doesn t recognize any more, that this is a list